covsirphy/cleaning/jhu_data.py

```diff
@@ -558,3 +558,64 @@ def records(self, country, province=None, start_date=None, end_date=None, popula
             raise SubsetNotFoundError(
                 country=country, country_alias=country_alias, province=province,
                 start_date=start_date, end_date=end_date, message="with 'Recovered > 0'") from None
+
+    def show_complement(self, country=None, province=None,
+                          start_date=None, end_date=None, population=None,
+                          interval=2, max_ignored=100, max_ending_unupdated=14):
+        """
+        To monitor effectivity and safety of complement on JHU subset,
+        we need to know what kind of complement was done for JHU subset
+        for each country (if country/countries specified) or for all countries.
+
+        Args:
+            country(str): country name or ISO3 code
+            province(str or None): province name
+            start_date(str or None): start date, like 22Jan2020
+            end_date(str or None): end date, like 01Feb2020
+            population(int or None): population value
+            interval (int): expected update interval of the number of recovered cases [days]
+            max_ignored (int): Max number of recovered cases to be ignored [cases]
+            max_ending_unupdated (int) : Max number of days to apply full complement,
+                 where ending recovered cases are not updated [days]
+
+        Returns:
+            pandas.DataFrame:
+                Index: reset index
+                Columns:
+                    - country(str): country name or ISO3 code
+                    - Monotonic_confirmed (str/bool): True if applied for confirmed cases or False otherwise
```

Comment:
I thought only `bool` will be returned and `str` will not be returned. Is this correct?